OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -86,6 +86,27 @@ public void PlayMusic(string track = null, LuaFunction func = null)
 				playlist.Play(musicInfo);
 		}
 
+		[Desc("Play track defined in music.yaml as background music." +
+			" If music is already playing use Media.StopMusic() to stop it" +
+			" and the background music will start automatically." +
+			" Keep the track empty to disable background music.")]
+		public void SetBackgroundMusic(string track = null)
+		{
+			if (!playlist.IsMusicAvailable)
+				return;
+
+			playlist.SetBackgroundMusic(string.IsNullOrEmpty(track) ? null : GetMusicTrack(track));
+		}
+
+		MusicInfo GetMusicTrack(string track)
+		{
+			var music = world.Map.Rules.Music;
+			if (!music.ContainsKey(track))
+				throw new LuaException("Track '{0}' is not defined in music.yaml!".F(track));
```

Comment:
#9488 makes `PlayMusic` generate a soft error (writing a message to lua.log) instead.